galsim/utilities.py

```diff
@@ -1016,7 +1016,7 @@ def lod_to_dol(lod, N=None):
             except TypeError:  # Value is not list-like, so broadcast it in its entirety.
                 out[k] = v
             except Exception:
```

Comment:
Something I think we're probably not so good at in GalSim currently is that whenever you catch generic excdption via either `except Exception:` or simply `except:`, you should first catch `KeybordInterrupt` and reraise.  Otherwise, Ctrl-C doesn't always interrupt the program the way you'd want.